packaging/rpm/crc.spec.in

```diff
@@ -65,7 +65,12 @@ make COMMIT_SHA=__COMMIT_SHA__ GO_EXTRA_LDFLAGS="-B 0x$(head -c20 /dev/urandom|o
 %install
 # with fedora macros: gopkginstall
 install -m 0755 -vd                     %{buildroot}%{_bindir}
+%ifarch x86_64
 install -m 0755 -vp %{gobuilddir}/src/%{goipath}/out/linux-amd64/crc %{buildroot}%{_bindir}/
+%endif
+%ifarch aarch64
+install -m 0755 -vp %{gobuilddir}/src/%{goipath}/out/linux-arm64/crc %{buildroot}%{_bindir}/
```

Comment:
Could this use `go env GOARCH` instead of hardcoding these `ifarch`?

Reply:
didn't find how to run that in the `install` section? https://community.nethserver.org/t/cross-compiling-multi-arch-rpms/18493 is also suggest same what we are using.

Reply:
Do you think https://github.com/kovyrin/rpmbuild/blob/master/golang/SPECS/golang.spec#L27-L38 is better what we are having now?

Reply:
> Do you think https://github.com/kovyrin/rpmbuild/blob/master/golang/SPECS/golang.spec#L27-L38 is better what we are having now?

Yes I'd prefer something like this, it's a lot easier to see what is arch-specific, and what is common.